test/babel.test.mjs

```diff
@@ -57,4 +69,16 @@ describe('htm/babel', () => {
 			}).code
 		).toBe(`var name="world",vnode={type:1,tag:"div",props:{id:"hello"},children:[{type:3,tag:null,props:null,children:null,text:"hello, "},name],text:null};`);
 	});
+
+	test('preserves placeholder-looking strings in attributes and text', () => {
```

Comment:
Maybe this test could be removed, as the old string replacement based approach is no longer used?

Reply:
yup.